README

```diff
@@ -24,25 +24,43 @@ For more information and details on how to contribute to `htop` visit [htop.dev]
 
 ## Build instructions
 
-This program is distributed as a standard GNU autotools-based package.
-
-Compiling `htop` requires the header files for `ncurses` (libncursesw*-dev). Install these and other required packages for C development from your package manager.
-
-Then, when compiling from a [release tarball](https://github.com/htop-dev/htop/releases), run:
-
+### Prerequisite
+List of build-time dependencies:
+*  `build-essential` standard GNU autotools-based
+*  `autoconf`
+*  `autotools`
+*  `ncurses` (libncursesw*-dev)
+
+**Note about `ncurses`:**
+> htop requires ncurses 6.0, the appropriate package is often enough still called libncurses5 (+-dev for headers). Also ncurses usually comes in two flavours:
+>* With Unicode support.
+>* Without Unicode support.
+>
+> This is also something that is usually reflected in the package name (the additional w).
+
+List of additional build-time dependencies (based on feature flags):
+*  `sensors`
+*  `hwloc`
+*  `libcap`
+
+Compiling `htop` requires the header files for `ncurses` . Install these and other required packages for C development from your package manager.
+
+**Debian/Ubuntu**
 ~~~ shell
-./configure && make
+sudo apt install libncursesw5-dev autotools-dev autoconf
 ~~~
```

Comment:
If we're going to this level of detail, I guess add here:

**Fedora/RHEL**
~~~ shell
sudo dnf install ncurses-devel autoconf automake
~~~